client/src/actors/client.rs

```diff
@@ -426,6 +499,11 @@ impl Receive<SHRequest> for Client {
                 let payload = self.read_from_store(vid.into());
 
                 if let Some(payload) = payload {
+                    println!(
```

Comment:
do we want to keep this?

Reply:
no sorry, I only inserted that for debugging purposes, I'll remove it